src/stories/index.stories.tsx

```diff
@@ -101,22 +103,61 @@ storiesOf('Dashboard', module)
     </CardContainer>
   ));
 
-// storiesOf('Dashboard/Transactions', module)
-//   .add('Entry (expense)', () => (
-//   <GlobalWrapper>
-//     <TransactionEntry />
-//   </GlobalWrapper>
-//   ))
-//   .add('Entry (income)', () => <TransactionEntry />);
-
-// storiesOf('Transactions', module)
-//   .addDecorator(fn => (
-//   <div style={{ width: '80vw', margin: '2em', background: theme.palette.primary.main }}>
-//   {fn()}
-//   </div>
-//   ))
-//   .add('Transaction page', () => <Transactions />)
-//   .add('Recurring options', () => <RecurringTransactionOptions/>);
+const expenseTx = {
```

Comment:
Consider using the generator we created, and rather overwrite parameters if you need to set them explicitly. This would make the code a bit DRYer.

Reply:
I figured this is just Storybook, and it looks like 💩 	already.